dallinger/recruiters.py

```diff
@@ -872,6 +903,42 @@ def external_submission_url(self):
         self.prolificservice.log_response(response)
         return response
 
+    def screen_out_allowed(self, participants: list[Participant]):
```

Comment:
I can't see this being called anywhere? You need to override `DevProlificService._req` to refer to this.

Reply:
Addressed this one. It is now called in `DevProlificService._req`. I also moved the `screen_out_allowed` method to `dallinger/prolific.py`, as it seems the correct place.